.github/workflows/tmt-tests.yml

```diff
@@ -49,55 +59,73 @@ jobs:
       environment_settings: '{"provisioning": {"post_install_script": "#!/bin/sh\nsudo sed -i s/.*ssh-rsa/ssh-rsa/ /root/.ssh/authorized_keys; echo 42; yum-config-manager --enable rhel-7-server-rhui-optional-rpms"}}'
       pull_request_status_name: "7to8-aws-e2e"
       variables: "RHUI=aws"
+    if: |
+      github.event.issue.pull_request
+      && ! startsWith(github.event.comment.body, '/rerun-sst')
+      && contains(fromJson('["OWNER", "MEMBER", "COLLABORATOR"]'), github.event.comment.author_association)
 
   call_workflow_tests_86to90_integration:
     needs: call_workflow_copr_build
     uses: ./.github/workflows/reuse-tests-8to9.yml
     secrets: inherit
     with:
       copr_artifacts: ${{ needs.call_workflow_copr_build.outputs.artifacts }}
-      tmt_plan_regex: "^(?!.*c2r)(?!.*sap)(?!.*7to8)(?!.*morf)"
+      tmt_plan_regex: "^(?!.*upgrade_plugin)(?!.*tier[2-3].*)(?!.*c2r)(?!.*sap)(?!.*7to8)(?!.*morf)"
+      variables: 'TARGET_RELEASE=9.0;TARGET_KERNEL=el9;RHSM_REPOS=rhel-8-for-x86_64-appstream-eus-rpms,rhel-8-for-x86_64-baseos-eus-rpms'
       pull_request_status_name: "8.6to9.0"
+    if: |
+      github.event.issue.pull_request
+      && ! startsWith(github.event.comment.body, '/rerun-sst')
+      && contains(fromJson('["OWNER", "MEMBER", "COLLABORATOR"]'), github.event.comment.author_association)
 
-  call_workflow_tests_87to91_integration:
+  call_workflow_tests_87to90_integration:
     needs: call_workflow_copr_build
     uses: ./.github/workflows/reuse-tests-8to9.yml
     secrets: inherit
     with:
       copr_artifacts: ${{ needs.call_workflow_copr_build.outputs.artifacts }}
-      tmt_plan_regex: "^(?!.*c2r)(?!.*sap)(?!.*7to8)(?!.*morf)"
-      variables: 'LEAPP_DEVEL_TARGET_PRODUCT_TYPE=beta;RHSM_SKU=RH00069;TARGET_RELEASE=9.1;TARGET_KERNEL=el9;RHSM_REPOS=rhel-8-for-x86_64-appstream-rpms,rhel-8-for-x86_64-baseos-rpms'
+      tmt_plan_regex: "^(?!.*upgrade_plugin)(?!.*tier[2-3].*)(?!.*c2r)(?!.*sap)(?!.*7to8)(?!.*morf)"
+      variables: 'TARGET_RELEASE=9.0;TARGET_KERNEL=el9;RHSM_REPOS=rhel-8-for-x86_64-appstream-eus-rpms,rhel-8-for-x86_64-baseos-eus-rpms'
       compose: 'RHEL-8.7.0-Nightly'
-      pull_request_status_name: "8.7to9.1"
+      pull_request_status_name: "8.7to9.0"
       tmt_context: "distro=rhel-8.7"
+    if: |
+      github.event.issue.pull_request
+      && ! startsWith(github.event.comment.body, '/rerun-sst')
+      && contains(fromJson('["OWNER", "MEMBER", "COLLABORATOR"]'), github.event.comment.author_association)
 
   call_workflow_tests_86to90_sst:
     needs: call_workflow_copr_build
     uses: ./.github/workflows/reuse-tests-8to9.yml
     secrets: inherit
     with:
       copr_artifacts: ${{ needs.call_workflow_copr_build.outputs.artifacts }}
-      tmt_plan_regex: "^(?!.*c2r)(?!.*sap)(?!.*7to8)(.*morf)"
+      tmt_plan_regex: "^(/plans/morf)(?!.*sap)"
+      variables: 'TARGET_RELEASE=9.0;TARGET_KERNEL=el9;RHSM_REPOS=rhel-8-for-x86_64-appstream-eus-rpms,rhel-8-for-x86_64-baseos-eus-rpms'
```

Comment:
@fernflower  in case of upgrade from 8.7, eus repositories does not exist. Possibly we should use just non-eus repos for upstream tests?

Reply:
Makes sense! Thanks!

Reply:
Now 8.7->9.0 upgrade is run with non-eus repos (L93).